streamlit_web.py

```diff
@@ -42,6 +43,7 @@ def main():
             [
                 "Local file system",
                 "AWS",
+                "File Upload",
```

Comment:
Since "File upload" is also from the Local file system, it could be confusing for users. Is there a way to have a logic to integrate these two? Or maybe change the naming, so it's clearer that one is for path input and another for file uploader.